pom.xml

```diff
@@ -39,7 +39,7 @@
         <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
         <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
         <url>https://github.com/${gitHubRepo}</url>
-        <tag>v1.42.0</tag>
+        <tag>${scmTag}</tag>
```

Comment:
Matching the archetype, and also needed to get an incremental build for testing.